test/cache-driver.js

```diff
@@ -231,7 +232,7 @@ suite('CacheDriver', function() {
 
         var otherAuthority = 'someOtherAuthority';
         var responseOptions = { authority : otherAuthority, mrrt : true, resource : responses[0].resource };
-        var differentAuthorityResponse = util.createResponse(responseOptions);
+        var differentAuthorityResponse = util.createResponse(responseOptions, 21);
```

Comment:
Magic number?  Why 21?  Also, could you add a comment to the definition of the util.createResponse function that explains what the iteration parameter was for.  It took a while to refresh my memory.  I should have doc'ed it in the first place.  Something like "iteration - createResponse will create a distinct token for each value of iteration and it will always return that same token for the same value of iteration.  This is useful when you need a non singleton set of tokens to test a cache scenario against."